docs-src/content/getting-started/_index.md

```diff
@@ -1,6 +1,10 @@
 +++
-title = "Getting started"
+title = "Getting Started"
 description = ""
 weight = 1
-alwaysopen = true
 +++
+
+This section is meant to get you started using Cubic Server.  It includes:
+
+* [a page](./quickstart) on using our web demo or our command-line interface, and
```

Comment:
should the links be visible as `Quickstart` or `Server Setup` rather than `a page` here?